Core/Utilities/CFG.py

```diff
@@ -878,17 +874,16 @@ def loadFromFile( self, fileName ):
     :param fileName: File name to load the contents from
     :return: This CFG
     """
-    if gZipEnabled and fileName.find( ".zip" ) == len( fileName ) - 4:
+    if fileName.find( ".zip" ) == len( fileName ) - 4:
```

Comment:
ah yes ! Cool ! I guess this also checks the integrity of the archive and so on ?